proto/fundraising/tx.proto

```diff
@@ -89,9 +88,11 @@ message MsgCreateBatchAuction {
   // auctioneer specifies the bech32-encoded address that creates the auction
   string auctioneer = 1;
 
-  // allowed_bidders specifies the bidders who are allowed to bid for the
-  // auction
-  repeated AllowedBidder allowed_bidders = 2;
+  // start_price specifies the starting price of the auction
+  string start_price = 2 [
```

Comment:
Does this need to be included? A batch auction does not need this start_price, even though BaseAuction has the field.

Reply:
Discussed this internally. We decided to have the `start_price` as an informative value for front-end.